config/Version.public.xcconfig

```diff
@@ -1,4 +1,4 @@
 VERSION_SHORT=18.6
 
 // Public long version example: VERSION_LONG=9.9.0.0
-VERSION_LONG=18.6.0.0
+VERSION_LONG=18.6.0.2
```

Comment:
Went from `.0` to `.2` because I shipped a build yesterday afternoon and, because of the timing, didn't end up submitting a PR for it.